src/plugins/console/server/lib/spec_definitions/json/generated/indices.simulate_index_template.json

```diff
@@ -0,0 +1,16 @@
+{
+  "indices.simulate_index_template": {
+    "url_params": {
+      "create": "__flag__",
+      "cause": "",
+      "master_timeout": ""
+    },
+    "methods": [
+      "POST"
+    ],
+    "patterns": [
+      "_index_template/_simulate_index/{name}"
```

Comment:
This file should also have an override specified for patterns:

```
    "patterns": [
      "_index_template/_simulate_index/{indices}"
    ]
```

Changing "name" to the special value "indices" will provide index completion for this endpoint:

<img width="655" alt="Screenshot 2020-07-22 at 11 55 19" src="https://user-images.githubusercontent.com/8155004/88163001-570a6c80-cc12-11ea-8e18-582d05cfefb5.png">

According to the docs a template name could also be given, but I think having the indices in this case is quite nice.

Reply:
Done.

Reply:
I'm not sure I see how this change is helpful. There's a relationship between an index template's index patterns and the names of indices, but there's no relationship between an index template's name and the names of indices. Am I missing something?

Reply:
Ohh never mind. The [docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-simulate-index.html) specify that this should be an index name not an index template name. So I understand the intention behind JL's suggestion and agree with it. 👍